src/utils/transforms.test.ts

```diff
@@ -50,4 +50,34 @@ describe('transforms', () => {
     expect(transformData[address1].isCoreToken).toBeFalsy()
     expect(transformData[address2].isCoreToken).toBeTruthy()
   })
+
+  it('should not have undefined properties', () => {
+    const address1 = '0x00400fcbf0816bebb94654259de7273f4a05c762'
+    const address2 = '0x452ef5a4bd00796e62e5e5758548e0da6e8ccdf3'
+    const mockTokensInfo = [
+      {
+        address: address1,
+        name: 'Test Token 1',
+        symbol: 'TT1',
+        decimals: 18,
+      },
+      {
+        address: address2,
+        name: 'Test Token 2',
+        symbol: 'TT2',
+        decimals: 18,
+        isFeeCurrency: true,
+      },
+    ]
+
+    const transformData = transformCeloTokensForRTDB(mockTokensInfo)
+
+    for (const address of Object.keys(transformData)) {
+      expect(
+        Object.values(transformData[address]).every(isNotUndefined),
```

Comment:
nit: perhaps it'd be the most robust just to declare the expected data in full? (in case expected properties are missing, this test would then fail)

Reply:
okay makes sense, happy for you to also leave it like this